api/src/router/routes/users.js

```diff
@@ -0,0 +1,53 @@
+//
+// Realm-o-Matic
+//
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Shelly Xue Han
+//
+
+'use strict';
+
+import { asyncMiddleware } from '@bcgov/common-nodejs-utils';
+import { Router } from 'express';
+import { ACCESS_CONTROL } from '../../constants';
+
+const router = new Router();
+
+/**
+ * Get the user authorization status
+ * TODO: more auth logic
+ */
+router.get(
+  '/auth/:userId',
+  asyncMiddleware(async (req, res) => {
+    // const { userId } = req.params;
+    const { roles } = req.query;
+    try {
+      let authorizationCode = ACCESS_CONTROL.NO_ROLE;
+      if (roles.includes(ACCESS_CONTROL.REVIEWER_ROLE))
```

Comment:
few things
- it appears roles is not validated to an array and so if the api was called with `?role=fooRole` this would throw and send the below error --- is this okay?
- if roles are supposed to be an array, the if/else if chain establishes a priority order of what authorization code to send, is this done on purpose?

Reply:
and maybe we should get this route tested